### ISSUE TITLE
p4test doesn't reject program with missing default action arguments

### ISSUE THREAD

**Issue by antoninbas** [p4lang/p4c#1557]
The P4_16 spec reads:

> Note that the specified default action must supply arguments for the control-plane bound parameters (i.e., the directionless parameters), since the action is synthesized at compilation time. 

However, the follow program is not rejected by p4test, even though table `t` doesn't provide a value for parameter `idx` of action `rewrite` in its `default_action` property:
```
#include <core.p4>
#include <v1model.p4>

struct H { };
struct M {
    bit<16> idx;
}

parser ParserI(packet_in pk, out H hdr, inout M meta, inout standard_metadata_t smeta) {
    state start { transition accept; }
}

action empty() { }

control IngressI(inout H hdr, inout M meta, inout standard_metadata_t smeta) {

    action drop() { smeta.drop = 1; }

    action rewrite(bit<16> idx) { meta.idx = idx; }

    table t {
        key = { meta.idx : exact; }
        actions = { rewrite; }
        default_action = rewrite;
    }

    apply {
        t.apply();
    }

};

control EgressI(inout H hdr, inout M meta, inout standard_metadata_t smeta) {
    apply { }
};

control DeparserI(packet_out pk, in H hdr) {
    apply { }
}

control VerifyChecksumI(inout H hdr, inout M meta) {
    apply { }
}

control ComputeChecksumI(inout H hdr, inout M meta) {
    apply { }
}

V1Switch(ParserI(), VerifyChecksumI(), IngressI(), EgressI(),
         ComputeChecksumI(), DeparserI()) main;
```

**Comment by mbudiu-vmw** [comment]
I just submitted #1556, but it may not handle this. I'd like to merge that one first, though.

**Comment by mbudiu-vmw** [comment]
I appended a fix for this to #1556.
